### ISSUE TITLE
Sessions not recorded

### ISSUE THREAD

**Issue by aprufa** [olab/Open-Labyrinth#116]
I just had an update installed V.3.3 and now sessions are no longer recorded. Several cases were run but nothing showed up in the session tab. Any ideas? 

Thanks
Adam

**Comment by topps** [comment]
Hmm, have not seen that before. Anyone else come across this? dt

**Comment by sergey-rud** [comment]
Hello @aprufa,

Could you provide Apache’s error log and contents of the directory `/www/application/logs` ?

**Comment by NataliaStathakarou** [comment]
I came across a session problem when installing 3.4, some of the sessions are displayed but not clickable in the session section (i see the report and username but i get an error when i click to view). I can get them though from the database but i am wondering what the problem might be?

**Comment by sergey-rud** [comment]
Hello @NataliaStathakarou ,
If you get "Oops something went wrong" error, details can be found in the directory path_to_open_labyrinth/www/application/logs/2017 or just press cmd+q to see error details.

Can you share error details with us?

**Comment by NataliaStathakarou** [comment]
Thanks @sergey-rud 
The error is:
ERROR: type: ErrorException; code: Notice; message: Undefined offset: 34; file: ../application/classes/controller/reportmanager.php; line: 326; browser: firefox; browser version: 53.0; resolution: 1920x1080;

2017-06-07 07:45:57 --- ERROR: ErrorException [ 8 ]: Undefined offset: 34 ~ APPPATH/classes/controller/reportmanager.php [ 326 ]

**Comment by topps** [comment]
Yes, I have been noticing that lately as well. Have not yet figured out a pattern to it. dt

**Comment by sergey-rud** [comment]
@NataliaStathakarou,
I added a fix to release-3.4.1 branch (https://github.com/olab/Open-Labyrinth/tree/release-3.4.1) and to release-candidate branch, so you should checkout release-3.4.1 branch to pull the changes. 

Or if you don't use Git, you can replace 
www/application/classes/controller/reportmanager.php file 
with this https://github.com/olab/Open-Labyrinth/blob/acfbdda2bb1654eac9850c826bec7b4dc866a35e/www/application/classes/controller/reportmanager.php file 
manually.
